addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -28,13 +28,13 @@
                 </t>
                 <div class="o-autogrow"/>
                 <div class="o_ChatWindowHeader_item o_ChatWindowHeader_rightArea">
-                    <t t-if="chatWindow.hasShiftLeft">
-                        <div class="o_ChatWindowHeader_command o_ChatWindowHeader_commandShiftLeft" t-att-class="{ 'o-mobile': env.messaging.device.isMobile }" t-on-click="_onClickShiftLeft" title="Shift left">
+                    <t t-if="chatWindow.hasShiftPrev">
+                        <div class="o_ChatWindowHeader_command o_ChatWindowHeader_commandShiftPrev" t-att-class="{ 'o-mobile': env.messaging.device.isMobile }" t-on-click.stop="_onClickShiftPrev" t-att-title="shiftPrevText">
```

Comment:
Do not add a stop propagation. Why do you need it actually?
And if there is a reason, use mark event handled helper methods.

Reply:
You made me remove a `stopPropagation` in the method, so I added it there.
The stopPropagation is needed as otherwise it's considered as a chat window header click and toggles the chat window.

Reply:
Use `markEventHandled` and `isEventHandled` where there is an issue.